ios/Demo-iOS/Sources/Views/SitePreparationView.swift

```diff
@@ -0,0 +1,330 @@
+            } else {
+                Text("Editor Dependencies Missing – the editor will need to load them when it starts")
+            }
+
+            Section("Feature Configuration") {
+                Toggle("Enable Native Inserter", isOn: $viewModel.enableNativeInserter)
+                Toggle("Enable Network Logging", isOn: $viewModel.enableNetworkLogging)
+
+                // TODO: Loading this from the server would allow us to validate Custom Post Type support
+                Picker("Post Type", selection: $viewModel.postType) {
+                    Text("Post").tag("post")
+                    Text("Page").tag("page")
+                }
+            }
+
+            if let editorConfiguration = viewModel.editorConfiguration {
+                Section("Editor Configuration Details") {
+                    KeyValueRow(key: "Site URL", value: editorConfiguration.siteURL.absoluteString)
+                    KeyValueRow(key: "API Root", value: editorConfiguration.siteApiRoot.absoluteString)
+                    KeyValueRow(key: "Supports Block Assets", value: editorConfiguration.shouldUsePlugins)
+                    KeyValueRow(key: "Supports Theme Styles", value: editorConfiguration.shouldUseThemeStyles)
+                }
+            }
+
+            self.preloadSection
+
+            if let error = viewModel.error {
+                Section("Error") {
+                    Text(error.localizedDescription)
+                }
+            }
+
+            Section {} footer: {
+                Button(action: {
+                    self.viewModel.buildAndLoadConfiguration(navigation: navigation)
+                }) {
+                    Spacer()
+                    Text("Start Editor")
+                    Spacer()
+                }.buttonStyle(.borderedProminent)
+            }
+        }
+    }
+
+    var preloadSection: some View {
+        Section {
+            Button("Prepare Editor") {
+                withAnimation {
+                    self.viewModel.prepareEditor()
+                }
+            }.disabled(self.viewModel.disableButtons)
+
+            Button("Prepare Editor ignoring cache") {
+                withAnimation {
+                    self.viewModel.prepareEditorFromScratch()
+                }
+            }.disabled(self.viewModel.disableButtons)
+
+            Button("Clear Preload Cache") {
+                withAnimation {
+                    self.viewModel.resetEditorCaches()
+                }
+            }.disabled(self.viewModel.disableButtons)
+        } header: {
+            Text("Local Caches")
+        } footer: {
+            if let progress = self.viewModel.loadingProgress {
+                ProgressView(value: progress.fractionCompleted)
+            }
+        }.task {
+            await self.viewModel.countAssetBundles()
+        }
+    }
+}
+
+@Observable
+class SitePreparationViewModel {
+
+    var enableNativeInserter: Bool = false
```

Comment:
Might make sense to default this to `true` given that is what the WordPress mobile app use. 


```suggestion
    var enableNativeInserter: Bool = true
```